mitodl/mitxpro
# Update UI for selecting products in B2B purchase form

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1081 

#### What's this PR do?
Replaces the modal select with more traditional selects, but implemented with react-select so we can have more flexibility modifying the components. 

#### How should this be manually tested?
Go to `/ecommerce/bulk` and try different options. It should look similar to the mockup in the issue.

#### Screenshots (if appropriate)
![Screenshot from 2019-09-11 09-52-20](https://user-images.githubusercontent.com/863262/64703235-0c5c8780-d47a-11e9-9a46-8be990768da2.png)

![Screenshot from 2019-09-11 09-53-01](https://user-images.githubusercontent.com/863262/64703245-0ff00e80-d47a-11e9-9993-dcb3ddec2fad.png)
